synology-packages.md

```diff
@@ -13,33 +13,32 @@ dateCreated: 2022-05-06T13:45:19.731Z
 - The Servarr Team now creates and maintains Synology Packages
 - Installation instructions are noted below for the specific DSM versions
 
-> Generally the existing SynoCommunity versions are likely not compatible with the Servarr versions without some hoop jumping. This message to be updated and clarfied in the future.
+> Generally the existing SynoCommunity versions are likely not compatible with the Servarr versions without some hoop jumping. This means it would be wise te delete the old package after doing a [backup of your database](https://wiki.servarr.com/en/radarr/faq#how-do-i-backuprestore-radarr). Which can be done through the web interface of the starr app.
 {.is-warning}
 
-- ToDo - add blurb/instructions for migrating from SynoComm package
 
 > SynoCommunity has a list of [NAS by Architecture](https://github.com/SynoCommunity/spksrc/wiki/Architecture-per-Synology-model) which will assist you in identifying the correct package.
 {.is-info}
 
 ## DSM 6.x
 
 - The Lidarr-Official, Prowlarr-Official, Radarr-Official, Readarr-Official, and Sonarr packages should _just work_. 
-- Note that for Sonarr Mono is not a requires additional package and is bundled within the package. 
+- Note that Sonarr requires additional package and is bundled within the package. 
 - Download the release of the application for your NAS's archetecture from [the Servarr Syno Package GitHub](https://github.com/Servarr/spksrc/releases) and [manually install the package](https://kb.synology.com/en-us/DSM/tutorial/How_to_install_applications_with_Package_Center#x_anchor_id6) via package manager.
 
 ## DSM 7.x
 
 - The Lidarr-Official, Prowlarr-Official, Radarr-Official, and Readarr-Official packages should _just work_ for most architectures.
-  - Note that a NAS with `comcerto2k` requires additional steps
+  - Note that a NAS with `comcerto2k` requires additional steps, [see below](https://wiki.servarr.com/en/synology-packages#bubblewrap-manual-step-recommended-option).
 - Note that the Sonarr package requires additional steps for **all** architectures.
-- Note that for Sonarr Mono is not a requires additional package and is bundled within the package.
+- Note that Sonarr requires additional package installation and is bundled within the package.
```

Comment:
Missing a weird here somehow

Reply:
Not sure what you mean? I thought the previous sentence was weird?

Reply:
But you need bubblewrap? Which is the additional package and i bundled in the zip?

Reply:
Line 33 covers that.

This line is to note that they don't need mono